package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "dumi-theme-antd",
+  "name": "dumi-theme-antd-web3",
```

这里的 name 我们先不改，方便在有需要的时候 merge 回原仓库，我发包的时候本地 `stash pop`;

```suggestion
  "name": "dumi-theme-antd",
```